app/workers/fast_track/hypertension_medication_request_data.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module FastTrack
+  class HypertensionMedicationRequestData
+    attr_accessor :response
+
+    def initialize(response)
+      @response = response
+    end
+
+    def transform
+      entries = response.body['entry']
+      filtered = entries.filter { |entry| entry['resource']['status'] == 'active' }
+      filtered.map { |entry| transform_entry(entry) }
+    end
+
+    private
+
+    def pick(keys, hash)
+      hash.select { |k, _| keys.include? k }.with_indifferent_access
+    end
+
+    def transform_entry(raw_entry)
+      entry = pick(%w[status medicationReference subject authoredOn note dosageInstruction], raw_entry['resource'])
+      result = pick(%w[status authoredOn], entry)
```

Comment:
Could we use slice here as well please?

Reply:
Fixed in https://github.com/department-of-veterans-affairs/vets-api/commit/fdca3cb38787b06570888b86353450f3412a5a15.

Reply:
I am still seeing `pick` here.

Reply:
Gotcha! My mistake